### PR TITLE
done: historypage, likevideopage, watchlaterpage

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
**Added alert Toast, playlist functionality:**

_Files to check:_
  - PlaylistCard
  - PlaylistModal
  - playlist-context
  - SinglePlaylistPage
  - PlaylistPage
    

    
Live link: https://deploy-preview-5--planet-player.netlify.app/

**Preview**: 
image

image

image
 
image

### CODE REVIEW

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Code looks fine and functionality is also working in preview.
After adding to watch later, there should be some toast/alert to notify it has been added, I clicked 2-3 times thinking it was not added.
Also, after adding video to playlist, you should close the modal.

**Review by prankurpandeyy — APPROVED** [review]

**Review by Naman-Saxena1 — APPROVED** [review]
Functionality is working fine, I suggest you add functionality of add to playlist by clicking on playlist label.
Also add cursor: pointer on create button

Capture3

File: src/App.css [hunk 1]
```diff
@@ -46,3 +46,42 @@
...
+  justify-content: space-between;
+  display: flex;
+  position: relative;
+  margin: 0 auto;
+  font-size: 2.5rem;
+  color: gray;
+  align-items: center;
+  grid-gap: 8px;
+  grid-gap: 8px;
+  gap: 8px;
```
Comment (SinchanaMS) [inline src/App.css]: Use variables for the colors.
gap property is repeated thrice. Please check.

File: src/App.js [hunk 2]
```diff
@@ -7,6 +8,11 @@ import { Footer } from "./components/Footer/Footer";
 import { Navigation } from "./components/Navigation/Navigation";
 import { VideoListing } from "./pages/video-listing/VideoListing";
 import { SingleVideoCard } from "./components/Card/SinglevideoCard/SingleVideoCard";
+import { LikedVideoPage } from "./pages/LikedVideoPage/LikedVideoPage";
+import { WatchLater } from "./pages/WatchlaterPage/WatchLater";
+import { HistoryPage } from "./pages/HistoryPage/HistoryPage";
+import { PlaylistPage } from "./pages/PlaylistPage/PlaylistPage";
+import { SinglePlaylistPage } from "./pages/SinglePlaylistPage/SinglePlaylistPage";
```
Comment (SinchanaMS) [inline src/App.js]: You can make use of barrels (a common index file) to import these files to make the code concise.

File: src/components/Card/PlaylistCard/PlaylistCard.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+console.log("from playlistpAGE",playlist)
+console.log("from playlistpAGE",videos)
+console.log("from playlistpAGE",videos.length)
+
+  const navigate = useNavigate();
+
+//   const { addToHistory, setHistory } = useHistoryContext();
+//   const { addToLikeVideo, setLikeVideo } = useLikeVideoContext();
+//   const { addPlaylists, addToPlaylist } = usePlaylistContext();
+//   const { addToWatchLater, setWatchLater } = useWatchLaterContext();
```
Comment (SinchanaMS) [inline src/components/Card/PlaylistCard/PlaylistCard.jsx]: Remove logs and unused commented code.

File: src/components/Card/SinglevideoCard/singlevideo.css [hunk 4]
```diff
@@ -30,7 +30,7 @@
     grid-gap: 0rem;
     gap: 0rem;
```
Comment (SinchanaMS) [inline src/components/Card/SinglevideoCard/singlevideo.css]: gap is repeated here as well.

**Review by SinchanaMS — APPROVED** [review]
The UI looks good.  Also liked how separate playlist toasts are shown for the creation of playlists and on video being added to them. Good work!

I found some repetitive properties in css files, especially gap. Please check.
Remove logs and unused commented code for better readability of code.

**Comment by Adarsh-Patel-Dev** [comment]
Added toast Alerts, playlist functionality

**Pull request merged**